topics/data-science/tutorials/python-plotting/tutorial.md

```diff
@@ -214,7 +147,7 @@ plt.ylabel('N')
 
 You can also change different parameters and customize the plot. 
 
-```python
```

Comment:
oh it's fine here! It was just for the question/solution boxes where there are some edge cases which are weird

So sorry for all the requests :(

Reply:
I removed the python tag from all the code blocks, I hope that it is not a problem

Reply:
No problem